### DISCUSSION TITLE
how do i draw outside of the canvas border?

### DISCUSSION THREAD

**Discussion by FBPanimations** [keiwando/toonsquid#619] [Q&A]
it doesn't let me draw farther than the edge of the canvas. i am pretty sure it is possible though because i see it done in the official  tutorial videos and the demo animation. i have been looking for like an hour but i cannot find a single tutorial that even mentions this feature. please how do i do this.
![IMG_5466](https://github.com/keiwando/toonsquid/assets/95978299/2bd25b4b-0806-4f7b-b26f-d0fc7b64057d)
![IMG_5468](https://github.com/keiwando/toonsquid/assets/95978299/c61782b5-8869-4446-870e-9dd8bdc21e08)
it is so fucking late

**Comment by andydjames** [comment]
Vectors can draw outside the canvas, rasters can't.

**Comment by SharpsterDraws** [comment]
I had the same issue and was going nut trying to resolve the issue. I converted all my Raster Brushes to Vector Brushes and now I only work with Vector art for my animations. I now love working in Vector using Toonsquid (I never thought I would work in vector - bad memories of Adobe Animate). 

Try switching your work flow to vector and I promise your workflow will become much easier. You can stilll throw in some raster images here and there where it’s needed.

**Reply by FBPanimations** [reply]
ty!!
